source/index.js

```diff
@@ -2,15 +2,22 @@ var Pixi = require("pixi.js")
 var Afloop = require("afloop")
 var Keyb = require("keyb")
 
-var WIDTH = 230, HEIGHT = 130
+import Player from "./player.js"
 
-var renderer = Pixi.autoDetectRenderer(WIDTH, HEIGHT)
+window.GAME_WIDTH = 230
+window.GAME_HEIGHT = 130
```

Comment:
So what I'm thinking, because global variables like this are generally frowned upon, is that we should make a file that both `index.js` and `player.js` can import, which has these constants in it.

```js
// Game.js
module.exports.WIDTH = 230
module.exports.HEIGHT = 130
```

```js
// index.js
var Game = require("./Game.js")
var renderer = Pixi.autoDetectRenderer(Game.WIDTH, Game.HEIGHT)
```

```js
// player.js
var Game = require("./Game.js")
...
    if((this.position.y + this.height) < Game.HEIGHT)
```